apps/angular/projection/src/app/ui/card/card.component.html

```diff
@@ -1,21 +1,14 @@
 <div
   class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3"
   [class]="customClass">
-  <img
-    *ngIf="type === CardType.TEACHER"
-    src="assets/img/teacher.png"
-    width="200px" />
-  <img
-    *ngIf="type === CardType.STUDENT"
-    src="assets/img/student.webp"
-    width="200px" />
+  <ng-content/>
 
   <section>
     <app-list-item
       *ngFor="let item of list"
-      [name]="item.firstname"
       [id]="item.id"
-      [type]="type">
+      (deleteItem)="delete($event)">
+      <ng-template [ngTemplateOutlet]="templateRef" [ngTemplateOutletContext]="{item: item}"/>
     </app-list-item>
```

Comment:
to make it even more customizable, the list item can be call inside the parent. Because what will happen if we want to create a card with list item without a delete button ?